testing_and_setup/compass/ocean/global_ocean/EC60to30/init/config_E3SM_coupling_files.ini

```diff
@@ -0,0 +1,73 @@
+# This is the custom config file specific to this test case.  Any values here
+# will override the defaults in defaults.ini  Users wishing to alter the
+# configuration should modify this file, rather than defaults.ini
+
+[main]
+nprocs = 1
+# the file name of the initial condition, possibly after spin-up
+initial_condition = ../../spin_up/simulation/restarts/restart.0001-21-01_00.00.00.nc
+
+[mesh]
+short_name = ${prefix}.E3SMv${e3sm_version}.${min_res}to${max_res}km.${levels}L.rev${mesh_version}
+prefix = EC
+long_name = Eddy Closure mesh for E3SM Version ${e3sm_version},
+            ${min_res}-${max_res}km resolution, ${levels} vertical levels
+description = MPAS Eddy Closure mesh with enhanced resolution around the equator
+              (30 km) and poles (35 km) with 60-km resolution at mid latitudes.
+e3sm_version = 2
+mesh_version = 001
+creation_date = autodetect
+min_res = 30
+max_res = 60
+max_depth = autodetect
+levels = autodetect
+runoff_description = <<<Spreading function described here>>>
+
+# The following options need to be added by the author
+author =
+institution =
```

Comment:
@mark-petersen (and others), I haven't come up with a good way to fill these in in an automated way.  It might be helpful, e.g. for regression testing, to have a command-line flag that supplies these.  Needs a little thought.